.cypress/integration/7_app_analytics.spec.js

```diff
@@ -351,12 +352,15 @@ describe('Viewing application', () => {
     cy.get('[data-test-subj="app-analytics-panelTab"]').click();
     cy.get('[aria-label="actionMenuButton"]').click();
     cy.get('[data-test-subj="editVizContextMenuItem"]').click();
-    suppressResizeObserverIssue();
     cy.get('[data-test-subj="superDatePickerShowDatesButton"]').should('contain', 'Last 24 months');
     cy.get('.euiTab[id="availability-panel"]').click();
-    cy.get('[title="Bar"]').click();
+
+    cy.wait(delay);
+    cy.get('[data-test-subj="comboBoxInput"]').click();
+    cy.focused().type('{downArrow}');
```

Comment:
duplicate to below?

Reply:
Was used to select TimeSeries as the third option by going down twice, adding a commit to make this more clear and select it in a better way. It previously selected Horizontal Bar which did not have the availability function.